### ISSUE TITLE
Redundant Title Text

### ISSUE THREAD

**Issue by Scott-James-Hurley** [UserOfficeProject/user-office-project-issue-tracker#1678]
In the Proposals and Assignments and Legacy Proposals tables On the FAP page, each row's title value has a redundant title HTML attribute. This is because the title attribute's value is the same as the displayed text. The title attribute should be used to provide advisory information. We can probably just remove it as it serves no purpose here.
